core/src/main/java/org/infinispan/eviction/impl/PassivationManagerImpl.java

```diff
@@ -99,14 +102,14 @@ public void passivateAll() throws PersistenceException {
       if (enabled && !skipOnStop) {
          long start = timeService.time();
          log.passivatingAllEntries();
-         long count = 0;
-         for (InternalCacheEntry e : container) {
-            count++;
-            if (trace) log.tracef("Passivating %s", e.getKey());
-            persistenceManager.writeToAllNonTxStores(marshalledEntryFactory.newMarshalledEntry(e.getKey(), e.getValue(),
-                                                                                               internalMetadata(e)), BOTH);
-         }
-         log.passivatedEntries(count, Util.prettyPrintTime(timeService.timeDuration(start, TimeUnit.MILLISECONDS)));
+
+         List<MarshalledEntry> entries = StreamSupport
```

Comment:
We could use the state transfer `chunkSize`. I tend to like using that value for stuff like this :)